src/nvim/diff.c

```diff
@@ -2017,17 +2017,24 @@ int diff_infold(win_T *wp, linenr_T lnum)
 }
 
 /// "dp" and "do" commands.
-///
-/// @param put
-void nv_diffgetput(int put)
+void nv_diffgetput(bool put, size_t count)
 {
   exarg_T ea;
-  ea.arg = (char_u *)"";
+  char buf[30];
+
+  if (count == 0) {
+    ea.arg = (char_u *)"";
+  } else {
+    vim_snprintf(buf, 30, "%ld", count);
```

Comment:
I won't be able to test this with a real compiler before next week - but based on [this](http://trac.wxwidgets.org/ticket/16596) it seems it is not supported in VS2013. The closest I found was [%Id](http://its.ffmpeg.org/ticket/3472).